fdbserver/include/fdbserver/DDRelocationQueue.h

```diff
@@ -32,4 +34,333 @@ struct IDDRelocationQueue {
 	;
 };
 
```

Comment:
Could you add comments for the top level structs/classes in this header file (at least `RelocateData` and `DDQueue`), so that a reader can tell what each of these is responsible for?